src/Uno.UI/UI/Xaml/UIElement.crossruntime.cs

```diff
@@ -131,6 +131,7 @@ private void OnElementLoaded()
 				{
 					if (resource is FrameworkElement resourceAsUIElement)
 					{
+						resourceAsUIElement.XamlRoot = XamlRoot;
```

Comment:
Same here, is it set back to `null` on Unloaded?

Reply:
Actually realized that should not happen - once an element is associated with a `XamlRoot`, it must stay there - 
![image](https://github.com/unoplatform/uno/assets/1075116/1cf65dd2-d69c-4318-8418-5bd5a1675d76)